ledger/participant-integration-api/src/main/scala/platform/apiserver/services/ApiVersionService.scala

```diff
@@ -53,12 +53,12 @@ private[apiserver] final class ApiVersionService private (
 
   private val featuresDescriptor =
     FeaturesDescriptor.of(
-      userManagement = Some(UserManagementFeature()),
+      userManagement = Some(UserManagementFeature(supported = true)),
       experimental = Some(
         ExperimentalFeatures(
           selfServiceErrorCodes =
-            Option.when(enableSelfServiceErrorCodes)(ExperimentalSelfServiceErrorCodes()),
-          staticTime = Option.when(enableStaticTime)(ExperimentalStaticTime()),
+            Some(ExperimentalSelfServiceErrorCodes(supported = enableSelfServiceErrorCodes)),
+          staticTime = Some(ExperimentalStaticTime(supported = enableStaticTime)),
```

Comment:
I'd suggest to not change this for the `ExperimentalSelfServiceErrorCodes` as the 1.18 versions of the test tool will use presence of the message to detect whether self-service error codes are returned.

Reply:
Good point.
Will rollback that chang as there's no easy way to adapt the compatibility tests.